unipipe/commands/generate.ts

```diff
@@ -49,16 +49,49 @@ function generateUuid() {
   console.log(uuid.generate());
 }
 
-function generateCatalog() {
+async function generateCatalog() {
   console.log(catalog);
+  const destinationDir = await Input.prompt({
+    message: "Pick a destination directory for the generated catalog file:",
+    default: "./",
+  });
+  const dir: Dir = {
+    name: destinationDir,
+    entries: [
+      { name: "catalog.yml", content: catalog },
+    ],
+  };
+  writeDirectory(dir);
 }
 
-function generateTransformHandler() {
+async function generateTransformHandler() {
   console.log(transformHandler);
+  const destinationDir = await Input.prompt({
+    message: "Pick a destination directory for the generated transform file:",
+    default: "./",
+  });
+  const dir: Dir = {
+    name: destinationDir,
+    entries: [
+      { name: "handler.js", content: transformHandler },
+    ],
+  };
+  writeDirectory(dir);
 }
 
-function generateGithubWorkflow() {
+async function generateGithubWorkflow() {
   console.log(githubWorkflow);
+  const destinationDir = await Input.prompt({
+    message: "Pick a destination directory for the generated github-workflow file:",
+    default: "./",
+  });
+  const dir: Dir = {
+    name: destinationDir,
+    entries: [
+      { name: "github-workflow.yml", content: githubWorkflow },
+    ],
+  };
+  writeDirectory(dir);
```

Comment:
s: Writing a function and calling it three times could make the code easier to adapt and maintain in the future.

Reply:
i have also similar idea about it. i will try to improve this 👍

Reply:
i added a new function 👍